src/suavelib/Suave.sol

```diff
@@ -246,13 +246,55 @@ library Suave {
         return abi.decode(data, (string));
     }
 
-    function randomUint256() internal returns (uint256) {
-        (bool success, bytes memory data) = RANDOM_UINT256.call(abi.encode());
+    function randomBytes(uint64 numBytes) internal returns (bytes memory) {
```

Comment:
You cannot modify this file. It gets synced from `suave-geth`. I would do a `src/Random.sol`.